frontend/src/components/Blog/Story.tsx

```diff
@@ -1,197 +1,186 @@
+    if (submittingBookmark) {
+      return <Spinner />;
+    }
+    if (!blog.bookmarkId) {
+      return (
+        <Tooltip message="Save">
+          <BookmarkIcon onClickIcon={bookmarkPost} className="w-10 h-10 p-2 cursor-pointer" />
+        </Tooltip>
+      );
+    }
+    return (
+      <Tooltip message="Unsave">
+        <BookmarkSolid onClickIcon={unbookmarkPost} className="w-10 h-10 p-2 cursor-pointer" />
+      </Tooltip>
+    );
+  };
+
+  return (
+    <div className="text-slate-500 py-2 items-center justify-between flex border-y border-slate-200">
+      <div className="text-sm">
+        <ClapButton
+          clapCount={blog?.claps?.length || 0}
+          handleClap={likeBlog}
+        />
+      </div>
+      <div className="flex justify-center items-center">
+        {determineBookmarkView()}
+        {isAuthor && (
+          <>
+            <Tooltip message="Edit">
+              <button
+                onClick={beginEditStory}
+                type="button"
+                name="edit-story"
+                className="focus:outline-none font-medium rounded-lg text-sm px-4"
+              >
+                <EditIcon />
+              </button>
+            </Tooltip>
+            <Tooltip message="Remove">
+              <button
+                onClick={deleteStory}
+                type="button"
+                name="delete-story"
+                className="focus:outline-none font-medium rounded-lg text-sm px-2"
+              >
+                <RemoveIcon />
+              </button>
+            </Tooltip>
+          </>
+        )}
+      </div>
+      <Modal
+        message={"Are you sure that you want to unbookmark this post?"}
+        openModal={openUnbookmarkModal}
+        onConfirm={onConfirmUnbookmark}
+        onCloseModal={() => setOpenUnbookmarkModal(false)}
+      />
+    </div>
+  );
 };
 
 const AuthorBox = ({
-	name,
-	details,
-	publishedDate,
+  name,
+  details,
+  publishedDate,
 }: {
-	name: string;
-	details: string | undefined;
-	publishedDate: string;
+  name: string;
+  details: string | undefined;
+  publishedDate: string;
 }) => (
-	<div className="p-4">
-		<div className="flex items-center gap-4 py-4">
-			<Avatar name={name || "Anonymous"} />
-			<div>
-				<div className="font-bold">{name || "Anonymous"}</div>
-				<div>
-					<span>{details ? details : "An artist at living. My work of art is my life."} </span> ·{" "}
-					<span className="text-sm text-slate-500">{publishedDate}</span>
-				</div>
-			</div>
-		</div>
-	</div>
+  <div className="p-4">
+    <div className="flex items-center gap-4 py-4">
+      <Avatar name={name || 'Anonymous'} />
+      <div>
+        <div className="font-bold">{name || 'Anonymous'}</div>
+        <div>
+          <span>{details ? details : 'An artist at living. My work of art is my life.'} </span> ·{' '}
+          <span className="text-sm text-slate-500">{publishedDate}</span>
+        </div>
+      </div>
+    </div>
+  </div>
 );
+
```

Comment:
It is difficult to figure out your actually changes because of the formatting changes. 
Can you rollback the linting changes please ?

I understand that we don't have a proper linting strategy in the project yet. I will create a new issue where linting can be addressed.